core/src/autogluon/core/metrics/quantile_metrics.py

```diff
@@ -0,0 +1,39 @@
+""" Metrics for quantile regression """
+
+import logging
+
+import numpy as np
+
+from . import make_scorer
+
+logger = logging.getLogger(__name__)
+
+
+def _pinball_loss(target_value, quantile_values, quantile_levels):
```

Comment:
maybe also support sample_weight and just take weighted mean at the end?

Reply:
You mean quantile_weight? right? Yes, I agree, probably need to support it to differently weight quantile levels.

Reply:
No sorry, I mean sample_weight as in sklearn metrics, eg. https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html

This weighs different datapoints differently, eg. if I have two datapoints with sample_weights 0.1 and 0.9, then the overall pinball_loss should more closely reflect the pinball_loss incurred for the latter datapoint. 

Re quantile_weight to different weight different quantile-levels, this would be nice to add too!  In fact this is the main metric used in gluonTS:  https://github.com/awslabs/gluon-ts/blob/295ae085967446df9ebcd337a08010403774824c/src/gluonts/mx/block/quantile_output.py#L24

Reply:
I've implmented both sample_weight and quantile_weight. but at this moment, only sample_weight will work. 
we need to get additional argument for quantile_weight,